frontend/pages/about/activism-network.vue

```diff
@@ -3,12 +3,11 @@
     class="text-light-text dark:text-dark-text bg-light-content dark:bg-dark-content"
   >
     <Head>
-      <Title>{{ title }}</Title>
+      <Title>{{ $t("pages.about.activism-network.title") }}</Title>
     </Head>
-    <h1 class="p-4">This will be a page about our partner organizations</h1>
+    <h1 class="p-4">{{ $t("pages.about.activism-network.description") }}</h1>
```

Comment:
I'll likely remove this one and other placeholder ones so we don't localize things that ultimately are going to be removed :)

Reply:
Do you think should I remove these labels for these pages and left them static for now?

Reply:
I'd say it'd likely be best, but I can also get this if need be 😊